### PR TITLE
Cache card details

### PR DESCRIPTION
**Pull request by Yiin**
Fixes https://github.com/ProtonMail/proton-vpn-settings/issues/290

### CODE REVIEW

File: containers/payments/Method.js [hunk 1]
```diff
@@ -13,9 +13,11 @@ import Bitcoin from './Bitcoin';
 const { CARD, PAYPAL, BITCOIN, CASH } = PAYMENT_METHOD_TYPES;
 
 const Method = ({ type, amount = 0, currency, onCard, onPayPal, method, methods, loading }) => {
-    const { card, updateCard, errors, isValid } = useCard();
+    const cache = useCache();
+    const { card, updateCard, errors, isValid } = useCard(cache.get('card'));
 
     useEffect(() => {
+        cache.set('card', card);
```
Comment (mmso) [inline containers/payments/Method.js]: Could we find another solution than using `cache` for this?

**Comment by vincaslt** [comment]
Why is is it needed to cache card details on our end? Browsers do that by default if they detect a card input field: 
image

File: containers/payments/Method.js (hunk 1 above)
Comment (Yiin) [reply containers/payments/Method.js]: I guess we could cache card data in a variable outside the component, but I'm not sure if this PR is even relevant to the project for the reasons @vincaslt mentioned.

File: containers/payments/Method.js [hunk 2]
```diff
@@ -12,8 +12,9 @@ import Bitcoin from './Bitcoin';
 
 const { CARD, PAYPAL, BITCOIN, CASH } = PAYMENT_METHOD_TYPES;
 
-const Method = ({ type, amount = 0, currency, onCard, onPayPal, method, methods, loading }) => {
-    const { card, updateCard, errors, isValid } = useCard();
+const Method = ({ type, amount = 0, currency, onCard, onPayPal, method, methods, loading, cardToUse }) => {
+    const fallbackCard = useCard();
```
Comment (mmso) [inline containers/payments/Method.js]: Why is `fallbackCard` needed now? Can't we force `card` to always be passed?

**Comment by emnproton** [comment]
Verified as fixed.

**Pull request merged**